releasenotes/README.md

```diff
@@ -0,0 +1,41 @@
+## Release notes
+
+Release notes use [https://docs.openstack.org/reno/latest/](reno)
+
+Release notes are stored in `releasenotes/notes` directory.
+
+Reno allows to generate release notes using files in the repository as opposed to generating from commit messages.
+This makes it easier to review and edit release notes.
+
+## Development flow
+
+When submitting a PR with some changes worthy of mentioning in the notes (new feature, bugfix, deprecation, update requirements),
+committer should add a new note file using `reno new <note_name>` or `make reno-new note=<note_name>`.
```

Comment:
I think if we pass `--edit` to `reno new` it opens the newly created file so that we can edit it there only and don't have to open the newly created file again.
I think we should mention that and it would be helpful.

Reply:
Not sure that would work with docker container, but I can test it.

Reply:
It is possible. Switched to that as this is most likely the intention of anyone running this command. Need to rebuild the build image though.

Reply:
`reno new` now has `--edit` flag by default.